packages/esm-patient-common-lib/src/workspaces/workspaces.ts

```diff
@@ -154,6 +160,16 @@ export function launchPatientWorkspace(name: string, additionalProps?: object) {
   }
 }
 
+export function launchPatientChartWithWorkspaceOpen(
+  patientUuid: string,
+  workspaceName: string,
+  additionalProps?: object,
+) {
+  changeWorkspaceContext(patientUuid);
+  launchPatientWorkspace(workspaceName, additionalProps);
+  navigate({ to: '${openmrsSpaBase}/patient/' + `${patientUuid}/chart` });
```

Comment:
@brandones what do you think of optionally specifying the target view or dashboard?

Reply:
Oh yeah, of course that should be possible! Let me add that.